src/AutoRest.CSharp/LowLevel/Generation/LowLevelClientWriter.cs

```diff
@@ -49,7 +50,7 @@ public void WriteClient(CodeWriter writer, LowLevelClient client)
             using (writer.Namespace(clientType.Namespace))
             {
                 writer.WriteXmlDocumentationSummary($"{client.Description}");
-                using (writer.Scope($"{client.Declaration.Accessibility} partial class {clientType:D}"))
+                using (writer.Scope($"{client.Declaration.Accessibility} partial class {clientType:D}", scopeDeclarations: client.Fields.ScopeDeclarations))
                 {
                     WriteClientFields(writer, client);
```

Comment:
After the `scopeDeclarations`, we can invert the sequence of `WriteClientFields` and `WriteConstructors`?